mqtt/mqtt-generic/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "mqtt-generic"
+version = "0.1.0"
+authors = ["Azure IoT Edge Devs"]
```

Comment:
I'm planning to get rid of `mqtt-broker-core` and move the code back to `mqtt-broker` soon. I think we did separation wrong previously. It should be
```
[mqtt-edgehub] --> [mqtt-broker] <-- [mqtt-generic]
```
instead of 
```
[mqtt-edgehub]  +--> [mqtt-broker-core] <-- [mqtt-generic]
        ^               ^ 
        |               |
[mqtt-broker] ----------+
```

Reply:
:+1: great, I like it!